adapters/repos/db/vector/compressionhelpers/parallel_iterator_test.go

```diff
@@ -69,7 +74,7 @@ func TestCompressedParallelIterator(t *testing.T) {
 		},
 	}
 
-	quantization := []string{"pq"}
+	quantization := []string{"pq", "bq"}
```

Comment:
Wow! Good catch. Running both was the whole reason I added this. Never realized we were only running one. Great to see that tests still pass!